src/message/messageActionSheet.js

```diff
@@ -274,11 +283,28 @@ export const showActionSheet = (
       }
     })();
   };
+  let title: string;
+  if (Array.isArray(params.message.display_recipient)) {
+    // This means the message is a private message.
+    title = `${_('Private message between')} `;
+    params.message.display_recipient.forEach(
+      (recepient: PmRecipientUser, idx: number, recepients: PmRecipientUser[]) => {
+        title += `${recepient.full_name}`;
+        if (idx < recepients.length - 2) {
+          title += ', ';
+        } else if (idx === recepients.length - 2) {
+          title += ' & ';
+        }
+      },
+    );
+  } else {
+    title = `#${params.message.display_recipient} > ${params.message.subject}`;
+  }
   showActionSheetWithOptions(
     {
       ...(isHeader
         ? {
-            title: `#${params.message.display_recipient} > ${params.message.subject}`,
+            title,
```

Comment:
The new code always computes the title, even when unnecessary, and leaves a mutable variable floating around. Extracting title computation to a new function (instead of directly into this one) would avoid both of these.